src/phosphor-rbmc-data-sync-mgr_main.cpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include <phosphor-logging/lg2.hpp>
+#include <sdbusplus/async/context.hpp>
+#include <sdbusplus/server/manager.hpp>
+#include <xyz/openbmc_project/DataSync/BMCData/common.hpp>
+
+int main()
+{
+    try
+    {
+        using BMCDataSync =
+            sdbusplus::common::xyz::openbmc_project::data_sync::BMCData;
+
+        sdbusplus::async::context ctx;
+        sdbusplus::server::manager_t objManager{ctx,
+                                                BMCDataSync::namespace_path};
+
+        // clang-tidy currently mangles this into something unreadable
+        // NOLINTNEXTLINE
+        ctx.spawn(
+            [](sdbusplus::async::context& ctx) -> sdbusplus::async::task<> {
+            ctx.request_name(BMCDataSync::interface);
```

Comment:
I was looking at another review where the request_name isn't in a spawn: https://gerrit.openbmc.org/c/openbmc/dbus-sensors/+/75329/15/src/LeakDetection/LeakDetectionManager.cpp#104

not sure which way is better now.

Reply:
I haven’t been too concerned about this now, but I’d prefer to revisit it later if we are persisting anything. Otherwise, we might encounter [this issue](https://github.com/ibm-openbmc/openpower-hw-isolation/blob/main/src/hardware_isolation_main.cpp#L44) that I discussed with Patrick a while ago.

In my opinion, there's no need to spawn here, but I’d prefer to revisit this if any issues arise related to the above.